webview-ui/src/components/settings/sections/AboutSection.tsx

```diff
@@ -45,6 +72,63 @@ const AboutSection = ({ version, renderSectionHeader }: AboutSectionProps) => {
 						{" • "}
 						<VSCodeLink href="https://cline.bot/">https://cline.bot</VSCodeLink>
 					</p>
+
+					{PLATFORM_CONFIG.type === PlatformType.VSCODE && (
+						<>
+							<h3 className="text-md font-semibold">Logs</h3>
+							<p className="text-sm text-(--vscode-descriptionForeground)">
+								Cline writes detailed logs to help diagnose issues. Each session creates a separate log file. If
+								you encounter an error, these logs can be shared with the Cline team for troubleshooting.
+							</p>
+
+							<div className="flex flex-col gap-2 max-w-md">
+								<Button
+									className="w-full whitespace-normal min-h-[32px]"
+									onClick={() => FileServiceClient.openLogFile({})}
+									variant="secondary">
+									Current Log
+								</Button>
+								<p className="text-xs text-(--vscode-descriptionForeground) -mt-1 ml-1">
+									View the log file for this session
+								</p>
+
+								<Button
+									className="w-full whitespace-normal min-h-[32px]"
+									onClick={() => FileServiceClient.openLogsFolder({})}
+									variant="secondary">
+									All Logs
+								</Button>
+								<p className="text-xs text-(--vscode-descriptionForeground) -mt-1 ml-1">
+									Browse all session logs in your file manager
+								</p>
+
+								{logsSize > 0 && (
+									<>
+										<Button
+											className="w-full mt-1 whitespace-normal min-h-[32px]"
+											onClick={() => setShowClearDialog(true)}
+											variant="danger">
+											Clear Logs ({formatSize(logsSize)})
+										</Button>
+										<p className="text-xs text-(--vscode-descriptionForeground) -mt-1 ml-1">
+											Permanently delete all log files
```

Comment:
Unsure about the "permanently" wording here. It is permanent until the next time you run Cline and we log something, and then the log files are back. I think the note you have about "can't be undone" is plenty.